app/views/admin_request/_info_request.html.erb

```diff
@@ -12,7 +12,12 @@
       <%= both_links(info_request.user) %>
       <%= arrow_right %>
       <%= both_links(info_request.public_body) %>,
-      <%= admin_date(info_request.updated_at, ago_only: true) %>
+
+      <% if defined?(sort_order) && sort_order =~ /created_at/ %>
+        <%= admin_date(info_request.created_at, ago: false) %>
+      <% else %>
+        <%= admin_date(info_request.updated_at, ago_only: true) %>
+      <% end %>
```

Comment:
Approach here differs from `app/views/admin_user/_user.html.erb` maybe we should look to be consistence?

Reply:
Yeah I just worked off the existing implementation in each case – don't know if there's a good reason that they were different. Don't think the consistency is critical at this point but agree could look at in future.